library/src/main/java/com/chuckerteam/chucker/internal/support/DateJsonAdapter.kt

```diff
@@ -11,8 +11,8 @@ class DateJsonAdapter {
     private val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
 
     @ToJson
-    fun toJson(date: Date): String = dateFormat.format(date)
+    fun toJson(date: Date?): String? = date?.let { dateFormat.format(it) }
 
     @FromJson
-    fun fromJson(date: String): Date = dateFormat.parse(date)!!
+    fun fromJson(dateString: String?): Date? = dateString?.let { dateFormat.parse(it) }
```

Comment:
Can you explain why you decided to do nullable parameter and return value here nullable?
In my previous comment I talked about throwing an exception, not trying to hide possible errors by making nullables and hiding everything behind `?.let`

Reply:
Removed code related to `Date` parsing as it is not used anywhere in the codebase.